apim/templates/api-configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
           authentication:
             type: basic
             users:
-              admin: adminadmin
+              admin: admin
```

Comment:
Why updating the password here?

Reply:
My bad, I changed the password to test and commit by mistake. I'll revert it.